yaptide/routes/common_sim_routes.py

```diff
@@ -101,7 +100,39 @@ def get(user: UserBaseModel):
 
         estimators: list[EstimatorModel] = db.session.query(EstimatorModel).filter_by(simulation_id=simulation.id).all()
         if len(estimators) == 0:
-            return yaptide_response(message="Results are unavailable", code=404)
+            if simulation.platform == "DIRECT":  # also CODE TO REMOVE
+                return yaptide_response(message="Results are unavailable", code=404)
+            # Code below is for backward compatibility with old method of saving results
+            # later on we are going to remove it because it's functionality will be covered
+            # by the post method
+            # BEGIN CODE TO REMOVE
```

Comment:
can you explain what it that about ?

Reply:
For now we luck the feature of self subscribtion of jobs running on SLURM to database. The code below was part of separate GET method of `/results/batch`, but it was moved as part of GET method of `/results` so that UI can use 1 endpoint for all results. Later on it will be just deleted, because desirably SLURM jobs will use POST method of `/results` to update themselves.